test/fixtures/full/main.tf

```diff
@@ -52,22 +52,22 @@ locals {
 }
 
 module "vpc" {
-  source       = "terraform-google-modules/network/google"
-  version      = "~> 0.4.0"
+  source = "git::https://github.com/terraform-google-modules/terraform-google-network.git?ref=master"
```

Comment:
Can we pin this to a specific version? 0.12.x compatibility was introduced in https://github.com/terraform-google-modules/terraform-google-network/commit/7ba1b231e0662f4eaf264e5ad4a7e7ba977a40cd